apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -70,13 +71,10 @@ static void ble_device_scanned_cb(ble_scanned_device *scanned_device)
 		scanned_device->conn_info.addr.mac[4],
 		scanned_device->conn_info.addr.mac[5]
 	);
+
 	if (g_scan_done == 0) {
```

Comment:
scanned_device is a parameter passed  by callback function and accessed.
I think it should be checked with NULL to avoid NULL pointer dereferencing.

Reply:
If 'scanned_device' is NULL, this callback function can't be called.
It already checked in `framework/src/ble_manager/ble_manager_state.c line:754`